submissions/factorio_draftsman/main.md

```diff
@@ -0,0 +1,200 @@
+* **Draftsman is uni-language.** Written in Python, this makes it exceptionally easy to install, simple to use, and gives the user access to the entirety of Python's vast packaging index. Chances are, you can do it in Python with Draftsman, regardless of *what* Factorio-related thing you're actually doing.
+* **Draftsman is easy to use.** Designed from the start to be simple and (most importantly) self-documenting, Draftsman allows you to manipulate Blueprints and Entities by auto-completable attribute and method. 
+* **Draftsman is well-documented.** Every function, method, attribute, and class is documented and crosslinked at it's [readthedocs site](https://factorio-draftsman.readthedocs.io/en/latest/?badge=latest). In addition, tutorials and supplementary material is provided, as well as a whole host of [example programs](https://github.com/redruin1/factorio-draftsman/tree/main/examples) to help illustrate how Draftsman works.
+* **Draftsman is stable.** A rigorous suite of tests ensure that Draftsman behaves predictably and correctly (or at least correct enough), and code coverage is 100%. Draftsman is verified to work on the latest versions of Python 2 and 3, and is compatible with all versions of Factorio greater than 1.0, with the metrics to prove it.
+* **Draftsman is descriptive.** Draftsman enforces "Factorio-safety" as a core philosophy, which means that if there is any change to a blueprint that would result in an error on import, an exception is thrown. Draftsman also tries to enforce "Factorio-correctness", which means that values that won't break but are otherwise nonsensical will raise warnings. Both errors and warnings are verbose, so that any problem with your script can be understood and resolved in seconds.
+* **Draftsman is close-to-source.** Draftsman bases all of its data off of [Wube's `factorio-data` repository](https://github.com/wube/factorio-data), which means that all entities are exactly as you would expect in game with no inconsistencies. This makes Draftsman up-to-date, makes updating Draftsman for future Factorio versions a breeze, and allows versioning control to monitor changes between Draftsman and Factorio if any future breakage occurs.
+* **Draftsman supports mods.** Draftsman emulates the [Factorio data lifecycle](https://lua-api.factorio.com/latest/Data-Lifecycle.html) directly, meaning that the same loading process that happens when you launch the game is mimicked with a single Draftsman function. In addition to ensuring absolute accuracy to Factorio, this also makes it compatible with custom mod prototypes that can be accessed in script as if they were any other vanilla entity.
+
+Draftsman also has support for custom `"EntityLike"` objects, most notably `Group` objects, that allow you to create custom constructs that can be inserted into blueprints for aid in clarity and compartmentalization. Again, *MAYBE* I got a little carried away.
```

Comment:
I think this paragraph can use an example of what you can group together.

Reply:
Added an example paragraph between the two.